docker-entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+sed -i "s|username|${username}|g" ${exposeConfigPath} && sed -i "s|password|${password}|g" ${exposeConfigPath}
+
+if [[ $# -eq 0 ]]; then
+    exec /src/expose serve ${domain} --port ${port} --validateAuthTokens
```

Comment:
this is looking good, but i would still have `serve ${domain} --port ${port} --validateAuthTokens` on `CMD` which makes it clear to be the default command, then if something is provided change it...

Reply:
Yes but problem is that entrypoint was `ENTRYPOINT ["/src/expose"]`and it breaks like in #165

Other solution is to totally remove entrypoint.

Reply:
i mean having this entrypoint the way you did, just move this portion to CMD, so your entrypoint wouldn't have this if and just call `exec /src/expose "$@" and at Dockerfile you would add `CMD serve ${domain} --port ${port} --validateAuthTokens`

Reply:
Oh I see. I'm going to test that.

Reply:
I see it and like I did it was because Docker run `CMD` through `/bin/sh -c` after `ENTRYPOINT`. So, it produces exec `/src/expose "/bin/sh -c CMD`. This is why I am checking the arguments rather than setting the `CMD`.

Reply:
hmm, right... i think there is a difference calling `CMD comand as string` and `CMD ["comand", "as", "array"]`... string does this and array does not, as far as i remember....

but this is good as it is, sorry for the confusion :+1: